components/UploadForm/index.js

```diff
@@ -90,27 +100,30 @@ export default function UploadForm({ initialCategories, onSubmit }) {
           type="text"
           value={inputCategoryValue}
           onChange={onChangeInput}
-          onBlur={() => setIsCategoryFound(false)}
           style={{
             outline: noCategoriesSelected && "red 2px solid",
           }}
```

Comment:
i think you could exchange this style with  styled-components `<StyledInput  isRed={noCategoriesSelected}>` and `const StyledInput =styled.input`
outline: ${(props) => (props.isRed &&  "red 2px solid" )}
``i hope this helps and is correct  i am not shure about the quotes after the && :)   but i think it is your only style till now and its only about mixing styles later